core/ingest/pom.xml

```diff
@@ -81,7 +82,31 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>com.upplication</groupId>
+			<artifactId>s3fs</artifactId>
+			<version>1.5.3</version>
+		</dependency>
+		<dependency>
+			<groupId>com.github.jsr203hadoop</groupId>
+			<artifactId>jsr203hadoop</artifactId>
+			<version>1.0.1</version>
+		</dependency>
 	</dependencies>
+	<dependencyManagement>
```

Comment:
not a big deal, but we try to consolidate all "dependencyManagement" blocks within the root parent pom

Reply:
...actually httpclient looks like it was already added to the root pom's dependencyManagement